From 71e88ddd54bd869271cdf37c349c1ad26d1c4b7c Mon Sep 17 00:00:00 2001
From: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date: Sat, 31 Dec 2011 18:38:12 -0200
Subject: [PATCH] Remove implicit const char*->QString cast from QDBusServer's constructor

Commit 5be6cf0a6e306ed3a51ed5ba89317b1317544eea introduced an implicit
cast from const char* to QString in QDBusServer's constructor, which
breaks the compilation of applications which use QtDBus when
QT_NO_CAST_FROM_ASCII is defined and clang is used.

Task-number: QTBUG-23398
Change-Id: Iaba66ab05ae6232f69a3981c2ca168f1ffd7bb05
Reviewed-by: Thiago Macieira <thiago.macieira@intel.com>
---
 src/dbus/qdbusserver.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/dbus/qdbusserver.h b/src/dbus/qdbusserver.h
index 900c652..f4f19f8 100644
--- a/src/dbus/qdbusserver.h
+++ b/src/dbus/qdbusserver.h
@@ -61,7 +61,7 @@ class Q_DBUS_EXPORT QDBusServer: public QObject
 {
     Q_OBJECT
 public:
-    QDBusServer(const QString &address = "unix:tmpdir=/tmp", QObject *parent = 0);
+    QDBusServer(const QString &address = QLatin1String("unix:tmpdir=/tmp"), QObject *parent = 0);
     virtual ~QDBusServer();
 
     bool isConnected() const;
-- 
1.7.4.msysgit.0

